### PR TITLE
test(e2e): add E2E tests for update-email page

### PR DESCRIPTION
**Pull request by sitek94**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Solves "pages/update-email" #51705

### CODE REVIEW

File: e2e/update-email.spec.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      'placeholder',
+      'camperbot@example.com'
+    );
+    await expect(submitButton).toHaveAttribute('type', 'submit');
+    await expect(submitButton).toContainText(
+      translations.buttons['update-email']
+    );
+  });
+
+  test('The page has sing out button', async () => {
```
Comment (wahmd) [inline e2e/update-email.spec.ts:46]: small typo with 'sign' spelling.

Comment (sitek94) [reply e2e/update-email.spec.ts]: thanks!

**Review by raisedadead — APPROVED** [review]
#### :sparkles: :sparkles: **Superbly Done!** :sparkles: :sparkles: 

LGTM.

Thanks for this pull request and for contributing to the codebase for the first time. We will be looking forward to more contributions in the future.

Happy contributing.

**Review by Ahmadkashif — APPROVED** [review]
LGTM!

**Pull request merged**